### ISSUE TITLE
LIVENODING 691 / Blender Every Angle Render Camera

### ISSUE THREAD

**Issue by enzyme69** [enzyme69/blendersushi#85]
![prtscr capture_46](https://user-images.githubusercontent.com/3952687/29695236-fbe2a96c-8984-11e7-8309-87260411efe2.png)
![prtscr capture_45](https://user-images.githubusercontent.com/3952687/29695235-fbe19676-8984-11e7-900f-a9ca39c0fe14.png)
![prtscr capture_44](https://user-images.githubusercontent.com/3952687/29695237-fbfde556-8984-11e7-9fb3-6dd193bba74d.png)

[sv_every_angle_cam_006_2017_08_25_00_56.zip](https://github.com/enzyme69/blendersushi/files/1250565/sv_every_angle_cam_006_2017_08_25_00_56.zip)

This is a slight modification of my older live noding video "Matrix Camera"
https://www.youtube.com/watch?v=Y88o-ewcGFs

Basically we can quickly instance multiple cameras that aim into 3D object using Animation Nodes.

**Comment by enzyme69** [comment]
Watch out on the Camera List index, my mistake is to use index camera 1-42, where it supposed to be 0-41.
